### PR TITLE
fixes #2442 :Disable notification of taxonomy download

### LINKED ISSUE #2442: Disable notification of taxonomy download
Disable notification of taxonomy download. 
I disabled the one on success. People are complaining about the one shown during the download. Is it possible to not show it ?

### PR DESCRIPTION
**Pull request by mustaqmustu**
## Related issues and discussion
#2442 
 
 ## Screen-shots, if any
ezgif com-video-to-gif(8)

 
 ## Checklist
 
Make sure you've done all the following (You can delete the checklist before submitting)
 
 - [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.
 - [x] Code is well documented
 - [x] Include unit tests for new functionality
 - [x] All user-visible strings are made translatable
 - [x] Code passes Travis builds in your branch
 - [x] If you have multiple commits please combine them into one commit by squashing them.
 - [x] Read and understood the contribution guidelines .

### CODE REVIEW

**Comment by deniger** [comment]
Some clean up:

1. Could you remove unused import ( see image)
2. remove the parameter Intent of the method doTask ( line 39) as not used.
3. Please could you remove commented codes line 66, 68,...( not related to this PR but in the same file...)
4. Remove unused i18n string: txtDownloading

image

**Comment by mustaqmustu** [comment]
> @mustaqmustu, is it possible to:
> 
>     1. Could you remove unused import ( see image)
> 
>     2. remove the parameter Intent of the method doTask ( line 39) as not used.
> 
>     3. Please could you remove commented lines 66, 68,...( not related to this PR but in the same file...)
> 
>     4. Remove unused i18n string: txtDownloading
> 
> 
> image
> 
> thanks

changes made :)

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoadTaxonomiesService.java [hunk 1]
```diff
@@ -82,12 +58,8 @@ private void doTask(Intent intent) {
...
                 .observeOn(AndroidSchedulers.mainThread())
                 .toCompletable()
-                //.doOnSubscribe(d -> view.showLoading())
                  .subscribe(() -> {
                     //view.hideLoading(false);
-                  //   builder.setContentText(getString(R.string.txtLoaded));
-                 //    builder.setOngoing(false);
-                 //    notificationManager.notify(17, builder.build());
                 }, e -> {
                    //  e.printStackTrace();
```
Comment (brandenfung) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoadTaxonomiesService.java]: If `onComplete` and `onError` is not needed, just call `subscribe()` with no params.

Also as mentioned by @deniger, could you remove the string `txtDownloading` in strings.xml .

**Comment by deniger** [comment]
@brandenfung could we accept this PR as is and I will fix the feedbacks ( i18n to remove and fix LoadTaxanomy) in my next PR ( I'll have to merge...)

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoadTaxonomiesService.java (hunk 1 above)
Comment (mustaqmustu) [reply app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoadTaxonomiesService.java]: changes made :)

**Comment by mustaqmustu** [comment]
@brandenfung I have made the required changes.Can u approve this now?

**Pull request merged**